src/Parser/Shaman/Restoration/Modules/Talents/CloudburstTotem.js

```diff
@@ -26,10 +36,46 @@ class CloudburstTotem extends Analyzer {
     if (spellId !== SPELLS.CLOUDBURST_TOTEM_HEAL.id) {
       return;
     }
+    if(this.cbtActive) {
+      this._createFabricatedEvent(event, 'removebuff');
+      this.cbtActive = false;
+    }
 
     this.healing += event.amount;
   }
 
+  on_byPlayer_cast(event) {
+    const spellId = event.ability.guid;
+
+    if (spellId !== SPELLS.CLOUDBURST_TOTEM_TALENT.id) {
+      return;
+    }
+
+    // Patch 7.3.5 added a buffer before CBT can collect healing after casting,
+    // this turns out to be around 200ms and causes it to not collect healing from 
+    // spells casted right before it, essentially removing pre-feeding.
+    // This adds those 200ms to it so you can visually see that the feeding starts later.
+    const manipulatedEvent = {...event};
+    manipulatedEvent.timestamp = manipulatedEvent.timestamp + DELAY_MS;
+
+    this._createFabricatedEvent(manipulatedEvent, 'applybuff');
+    this.cbtActive = true;
+  }
+
+  _createFabricatedEvent(event, type) {
+    this.owner.fabricateEvent({
+      ...event,
+      ability: {
+        ...event.ability,
+        guid: SPELLS.CLOUDBURST_TOTEM_TALENT.id,
+      },
+      type: type,
+      targetID: event.sourceID,
+      targetIsFriendly: event.sourceIsFriendly,
+      __fabricated: true,
```

Comment:
`fabricateEvent` already sets `__fabricated`, so you don't have to do this here as well